### PR TITLE
[DRUP-508] Access is required to view rate plans.

### PR DESCRIPTION
**Pull request by Jaesin**

### CODE REVIEW

File: tests/src/Kernel/Controller/PackageControllerKernelTest.php [hunk 1]
```diff
@@ -54,12 +55,17 @@ protected function setUp() {
     $this->installConfig([
       'user',
     ]);
+    // Install user 0 and user 1.
+    \user_install();
```
Comment (minnur) [inline tests/src/Kernel/Controller/PackageControllerKernelTest.php:59]: Maybe remove `slash` at the beginning of the function name ? Is it really needed ?

File: tests/src/Traits/ApigeeMonetizationTestTrait.php [hunk 2]
```diff
@@ -144,7 +144,7 @@ protected function createAccount(array $permissions = [], bool $status = TRUE, s
       'first_name' => $this->randomMachineName(),
       'last_name' => $this->randomMachineName(),
       'name' => $this->randomMachineName(),
-      'pass' => user_password(),
+      'pass' => \user_password(),
```
Comment (minnur) [inline tests/src/Traits/ApigeeMonetizationTestTrait.php:147]: Is there a reason for a slash at the beginning of the function ?

**Review by minnur — COMMENTED** [review]
Few small questions/clarifications.

File: tests/src/Traits/ApigeeMonetizationTestTrait.php (hunk 2 above)
Comment (Jaesin) [reply tests/src/Traits/ApigeeMonetizationTestTrait.php:147]: When the user module isn't enabled yet, php is looking for `Drupal\Tests\apigee_m10n\Traits\user_password()`.

**Review by minnur — APPROVED** [review]
Looks good.

**Pull request merged**